### PR TITLE
Updating doc content

### PR DESCRIPTION
**Pull request by dowen12**
*Issue #, if available:*

*Description of changes:*
Doc content update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .specific/_qs_deploy_links.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[cols=",]
+|===
+|http://qs_launch_link[Deploy {partner-product-name} into a new VPC on AWS]
```
Comment (andrew-glenn) [inline .specific/_qs_deploy_links.adoc:3]: Shouldn't these be attributes? Can we placeholder them to https://example.com/

?

File: .specific/_settings.adoc [hunk 2]
```diff
@@ -4,5 +4,6 @@
 :doc-month: May
 :doc-year: 2020
 :partner-contributors: John Doe and Jane Doe
-:quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + \
-Joe Jones - Technical Product Manager, AWS
+:quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + Joe Jones - Technical Product Manager, AWS
+:deployment_time: 30 minutes / 1 hour
+:default_deployment_region: us-east-1
```
Comment (andrew-glenn) [inline .specific/_settings.adoc:9]: I'm not sure we want this as a setting.

File: .specific/_settings.adoc [hunk 3]
```diff
@@ -4,5 +4,6 @@
 :doc-month: May
 :doc-year: 2020
 :partner-contributors: John Doe and Jane Doe
-:quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + \
-Joe Jones - Technical Product Manager, AWS
+:quickstart-contributors: Jim Smith - AWS Global Partner SA, AWS + Joe Jones - Technical Product Manager, AWS
+:deployment_time: 30 minutes / 1 hour
```
Comment (andrew-glenn) [inline .specific/_settings.adoc:8]: I'm not sure we want this here. If we're going to average our successful tests, sure, but that should be an embedded value somewhere. We should directly control that via automation, not manual user input

File: .specific/faq_troubleshooting.adoc [hunk 4]
```diff
@@ -1,5 +1,7 @@
 // Add any tips or answers to anticipated questions. This could include the following troubleshooting information. If you don’t have any other Q&A to add, change “FAQ” to “Troubleshooting.”
 
+== FAQ
```
Comment (andrew-glenn) [inline .specific/faq_troubleshooting.adoc:3]: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

Search for '[qanda]'; I think that's a better style for these questions

File: _layout_cfn.adoc [hunk 5]
```diff
@@ -4,7 +4,7 @@
 == _{partner-product-name}_ on the AWS Cloud
 :!toc:
 [.text-left]
-ifdef::foo[include::../{includedir}/introduction.adoc[]]
+include::../{includedir}/introduction.adoc[]
```
Comment (andrew-glenn) [inline _layout_cfn.adoc:7]: Thanks! :)

File: .specific/_qs_deploy_links.adoc (hunk 1 above)
Comment (dowen12) [reply .specific/_qs_deploy_links.adoc:3]: So there are QS's that will have more than just 2 options for deployment. If thats the case, and this section lies in a 'common' doc we are not able to update it, hence pulling it out to its own doc

File: .specific/faq_troubleshooting.adoc (hunk 4 above)
Comment (dowen12) [reply .specific/faq_troubleshooting.adoc:3]: Potentially. We need to check what the Amazon style is for Q&A based questions. Are numbers allowed, does it actually need to resemble Q: A: etc

**Pull request merged**